aws-sso-assignment/aws-sso-assignment.json

```diff
@@ -0,0 +1,100 @@
+{
+  "typeName": "AWS::SSO::Assignment",
+  "description": "Resource Type definition for SSO assignmet",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-sso/aws-sso-assignment",
+  "properties": {
+    "InstanceArn": {
+      "description": "The sso instance that the permission set is owned.",
+      "type": "string",
+      "pattern": "arn:aws:sso:::instance/(sso)?ins-[a-zA-Z0-9-.]{16}",
```

Comment:
[should un-hardcode the `aws` partition](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arns-syntax) eventually in order to work in [other partitions](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants)